doc-content/enterprise-only/fuse/fuse-con.adoc

```diff
@@ -20,11 +20,21 @@ You can install {FUSE_LONG} on a separate instance of {EAP_LONG} and integrate i
 
 [IMPORTANT]
 ====
-For the version of {FUSE_LONG} that {PRODUCT} {ENTERPRISE_VERSION} supports, see 
+For the version of {FUSE_LONG} that {PRODUCT} {ENTERPRISE_VERSION} supports, see
 ifdef::PAM[]
 https://access.redhat.com/articles/3405381[{PRODUCT} 7 Supported Configurations].
 endif::[]
 ifdef::DM[]
 https://access.redhat.com/articles/3354301[{PRODUCT} 7 Supported Configurations].
 endif::[]
 ====
+
+[NOTE]
+====
+You can also install {FUSE_LONG} on SpringBoot. {PRODUCT} provides no special integration for this scenario. However, your application that runs on {FUSE_LONG} on SpringBoot can use the `kie-server-client` library to communicate with {PRODUCT} services that run on a {KIE_SERVER}. The following limitations apply, because of library version conflicts between {FUSE_LONG} and {PRODUCT}:
+
+* You cannot use the JSON marshalling format, because of incompatible versions of Jackson.
```

Comment:
```suggestion
* You cannot use `MarshallingFormat.JSON`, because of incompatible versions of Jackson.
```
I think this is better because Fuse users can still use JSON for other purposes outside of `kie-server-client`. `MarshallingFormat.JSON` is clear so they can understand that simply they cannot use `MarshallingFormat.JSON` configuration of `kie-server-client`.